CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 **This project is in Release Candidate stage.**
 
 ### Enhancements
-* None
+* Added values to items of `SyncResolveErrorCode` ([#1131](https://github.com/realm/realm-dart/pull/1131).
```

Comment:
this could be reworded cause now it is not clear what the improvement is.

Reply:
Is this better? "Added specific codes to `SyncResolveErrorCode` enum`s items."

Reply:
We should clarify to a developer what the enhancement is.

Reply:
Moved to Internal section.